src/MBC_UserImport_Source_Niche.php

```diff
@@ -264,15 +264,17 @@ public function process()
         if (empty($existing['email'])) {
             $this->addEmailSubscriptionSettings($this->importUser, $payload);
         }
-    
+
         // Drupal user
         $this->mbcUserImportToolbox->checkExistingDrupal(
             $this->importUser,
             $existing
         );
         if (empty($existing['drupal-uid'])) {
-            $northstarUser
-                = $this->mbToolbox->createNorthstarUser((object) $this->importUser);
+            $importUser = (object) $this->importUser;
```

Comment:
I know this isn't related to this PR, but I think it'd make sense to update this MB Toolbox method to accept an associative array in the future. Seems a bit silly to have to cast it to an object.